test/InterversionTests/SmugglerTests.cs

```diff
@@ -23,9 +23,13 @@ namespace InterversionTests
 {
     public class SmugglerTests : InterversionTestBase
     {
-        const string Server42Version = "4.2.102-nightly-20200415-0501";
+        const string Server42Version = "4.2.124-nightly-20230112-0944";
```

Comment:
Why do we need to bump the 4.2 server version?

Reply:
Because the previous doesn't include the latest changes (which fix some of these tests :) )